tests/integrations/test_dsm.py

```diff
@@ -27,20 +41,21 @@ def test_dsm_kafka(self):
         # checkpoint strings (e.g. service:foo)
         # There is currently no FNV-1 library availble for node.js
         # So we are using a different algorithm for node.js for now
-        if context.library == "nodejs":
-            producer_hash = 2931833227331067675
-            consumer_hash = 271115008390912609
-        else:
-            producer_hash = 4463699290244539355
-            consumer_hash = 3735318893869752335
+        language_hashes = {
+            "nodejs": {"producer": 2931833227331067675, "consumer": 271115008390912609,},
```

Comment:
This difference is fine IFF the hash isn't computed elsewhere.  Basically, we must guarantee:

1. DSM isn't computing the hash in the backend
2. Other client libraries aren't computing the hash

Can we:
1. Add a comment to this affect (part of this PR)
2. Figure out if there's an efficient hash which _is_ available across all languages (separately)

I want (2) because we may end up pre-computing hashes for destinations (i.e. A->B, A will compute "I am starting an attempt to send to B", at which point we'll want the same hash values to be generated).

Reply:
Also, it looks like `dd-trace-py` "just implements" the hashing algorithm; I wonder if we can do the same in NodeJS (outside this PR)